android/src/cg/group4/util/notification/AlarmReceiver.java

```diff
@@ -0,0 +1,45 @@
+package cg.group4.util.notification;
+
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Color;
+import android.support.v4.app.NotificationCompat;
+import cg.group4.android.AndroidLauncher;
+import cg.group4.android.R;
+
+/**
+ * A receiver for the alarm to display the notification.
+ */
+public class AlarmReceiver extends BroadcastReceiver {
+    /**
+     * Times for the vibration and led blinking.
+     */
+    protected final int cOnTime = 500, cOffTime = 200;
+
+    @Override
+    public void onReceive(final Context context, final Intent intent) {
+        Intent resultIntent = new Intent(context, AndroidLauncher.class);
+
+        PendingIntent pendingIntent =
+                PendingIntent.getActivity(context, 0, resultIntent, PendingIntent.FLAG_ONE_SHOT);
+
+        NotificationCompat.Builder notificationBuilder =
```

Comment:
You might want to split this to a separate method (as it is a behaviour on its own), for readability.